### PR TITLE
Apply uv transform in the prepass

### PR DESCRIPTION
**Pull request by DGriffin91**
# Objective

- The UV transform was applied in the main pass but not the prepass.

## Solution

- Apply the UV transform in the prepass.

## Testing

- The normals in my scene now look correct when using the prepass.

### CODE REVIEW

File: crates/bevy_pbr/src/render/pbr_prepass.wgsl [hunk 1]
```diff
@@ -29,6 +29,11 @@ fn fragment(
 #endif
 
     var out: prepass_io::FragmentOutput;
+    
+#ifdef VERTEX_UVS
+    let uv_transform = material.uv_transform;
+    let uv = (uv_transform * vec3(in.uv, 1.0)).xy;
+#endif
```
Comment (IceSentry) [inline crates/bevy_pbr/src/render/pbr_prepass.wgsl:36]: This is only used for the normal_prepass right? If so, it should probably be behind the ifdef of the normal prepass

Comment (DGriffin91) [reply crates/bevy_pbr/src/render/pbr_prepass.wgsl]: Correct. Good catch.

**Review by IceSentry — APPROVED** [review]

**Review by JMS55 — APPROVED** [review]

**Review by Elabajaba — APPROVED** [review]

**Comment by alice-i-cecile** [comment]
@DGriffin91 once the merge conflicts are resolved I'll merge this.

**Comment by DGriffin91** [comment]
@alice-i-cecile done. Also noticed the indentation in the adjacent area wasn't correct and fixed it.

**Comment by alice-i-cecile** [comment]
Oh no there's more :( Sorry!

**Pull request merged**